lockbox-ios/Common/Extensions/String+.swift

```diff
@@ -16,6 +16,13 @@ extension String {
         return shaData.base64URLEncodedString()
     }
 
+    func titleFromHostname() -> String {
+        return self
+                .replacingOccurrences(of: "http://", with: "")
```

Comment:
`replacingOccurrences` looks for anywhere within the string?

I think it would be best to limit to the start of the string.